schemas/session-replay/_common-wireframe-schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "_common-wireframe-schema.json",
+  "title": "CommonWireframe",
+  "type": "object",
+  "description": "Schema of common properties for Wireframe events type.",
+  "required": [
+    "id",
+    "x",
+    "y",
+    "width",
+    "height",
+    "type"
+  ],
+  "properties": {
+    "id": {
+      "type": "integer",
+      "description": "Defines the unique ID of the wireframe. This should be persistent throughout the view lifetime.",
+      "readOnly": true
+    },
+    "x": {
+      "type": "integer",
+      "description": "The position on X axis of the UI element in absolute coordinates. The anchor point will always be the top-left corner of the wireframe.",
+      "readOnly": true
+    },
+    "y": {
+      "type": "integer",
+      "description": "The position on Y axis of the UI element in absolute coordinates. The anchor point will always be the top-left corner of the wireframe.",
+      "readOnly": true
+    },
+    "width": {
+      "type": "integer",
+      "description": "The width in pixels of the UI element normalised based on the device DPI.",
```

Comment:
> normalised based on the device DPI.

**question:** What does that mean?

Reply:
Good point, I am going to detail more in the description.